dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/meta/ChildConnector.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2022-2024 Google LLC
+ * Copyright 2013-2021 CompilerWorks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.edwmigration.dumper.application.dumper.connector.meta;
+
+import com.google.edwmigration.dumper.application.dumper.connector.Connector;
+import com.google.edwmigration.dumper.application.dumper.task.Task;
+import javax.annotation.Nullable;
+
+public interface ChildConnector extends Connector {
+  @Nullable
+  default Task<?> createInitializerTask() {
```

Comment:
I'd prefer a non-null Optional in this context.

Reply:
Done.